scenarios/remote_call_rate/src/main.rs

```diff
@@ -45,12 +45,20 @@ fn agent_behaviour(
                 let admin_client = AdminWebsocket::connect(admin_ws_url, reporter.clone()).await?;
                 // No more agents available to call, get a new list.
                 // This is also the initial condition.
-                let mut new_peer_list = admin_client
+                let agent_infos_encoded = admin_client
                     .agent_info(None)
                     .await
-                    .context("Failed to get agent info")?
+                    .context("Failed to get agent info")?;
+                let mut agent_infos = Vec::new();
+                for info in agent_infos_encoded {
+                    let a = kitsune2_api::AgentInfoSigned::decode(
+                        &kitsune2_core::Ed25519Verifier,
+                        info.as_bytes(),
+                    )?;
+                    agent_infos.push(AgentPubKey::from_raw_32(a.agent.to_vec()))
```

Comment:
This conversion is defined in `holo_hash` behind a feature, should be possible to get that here and not duplicate the logic of this conversion

Reply:
[fixed](85dca7894b9a48407b87a78e656f5070346b5aea)